unitary/alpha/qubit_effects_test.py

```diff
@@ -34,6 +34,36 @@ def test_superposition():
     assert board.circuit == cirq.Circuit(cirq.H(cirq.NamedQubit("t")))
 
 
+def test_move():
+    board = alpha.QuantumWorld()
+    piece1 = alpha.QuantumObject("a", 1)
+    piece2 = alpha.QuantumObject("b", 0)
+    board.add_object(piece1)
+    board.add_object(piece2)
+    alpha.Move()(piece1, piece2)
+    expected_circuit = cirq.Circuit()
+    a = cirq.NamedQubit("a")
+    b = cirq.NamedQubit("b")
+    expected_circuit.append(cirq.X(a))
+    expected_circuit.append(cirq.SWAP(a, b))
+    assert board.circuit == expected_circuit
```

Comment:
Can you also test expected results too?  I hit this when my circuit was wrong before.

For instance,

```
results = board.peek([piece1, piece2], count =100)
assert all(result == [1,0] for result in results)
```

Same for below.

Reply:
I added two tests of `results` for both operations: One before the operation and one after the operation.

Reply:
What is the next step?

Reply:
Next step is to merge it into main, which I did for you.  Thanks for contributing!